libtransmission/peer-mgr.cc

```diff
@@ -2366,6 +2366,7 @@ void tr_peerMgr::bandwidth_pulse()
     for (auto* const tor : torrents_)
     {
         tor->do_idle_work();
+        tor->do_magnet_idle_work();
```

Comment:
Can we make `do_magnet_idle_work()` a private `tr_torrent` method and, instead of calling it here, call it from `tr_torrent::do_idle_work()`?

Reply:
39e05a8